src/main/java/org/folio/dao/invoice/InvoicePostgresDAO.java

```diff
@@ -105,11 +106,13 @@ public Future<DBClient> deleteInvoice(String id, DBClient client) {
     Promise<DBClient> promise = Promise.promise();
     client.getPgClient().delete(client.getConnection(), INVOICE_TABLE, id, reply -> {
       if (reply.failed()) {
+        log.error("Invoice deletion with id={} failed", id, reply.cause());
         handleFailure(promise, reply);
       } else {
         if (reply.result().rowCount() == 0) {
           promise.fail(new HttpException(Response.Status.NOT_FOUND.getStatusCode(), "Invoice not found"));
```

Comment:
Do we need to put logging in here?

Reply:
No, I am thinking to remove mostly if else block log.error because we will have log.error in catch exceptions which can catch failed cases. But regarding if else not found log.warn I think it needs

Reply:
in this case we don't have catch or something to put log.error somewhere in this method. in this class in if(reply.failed()) mostly used log.error() mybe, Can I use it in this case?